enrich/package.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
I think it makes sense to unify our two enrichment scripts into one package. Should we merge my PR, rebase this, and refactor?

Reply:
I think this is the best way to do it yes :)